typescript/sdk/src/token/checker.ts

```diff
@@ -126,6 +103,29 @@ export class HypERC20Checker extends ProxiedRouterChecker<
         this.addViolation(violation);
       }
     }
+
+    // Check all actual decimals are consistent
```

Comment:
if this needs to live here as opposed to other places, a comment would be useful